src/input/src/Input.tsx

```diff
@@ -1107,6 +1110,16 @@ export default defineComponent({
           )}
           {!this.pair &&
             resolveWrappedSlot($slots.suffix, (children) => {
+              const passwordVisibleIcon = this.passwordVisibleIcon ? (
+                h(this.passwordVisibleIcon)
```

Comment:
I think make it a slot is a better way. For example

```
// from src/_utils
resolveSlot(this.$slots.['password-visible'], () => [<EyeIcon />])
```

Reply:
Since it's more convient to compose.

Reply:
awesome😊